src/backroom/pspm_scr2ledalab.m

```diff
@@ -101,23 +101,15 @@
 
 % check output file & save data
 %--------------------------------------------------------------------------
-if exist(outfile, 'file') == 2 && options.overwrite ~= 1
-    if feature('ShowFigureWindows')
-        msg = ['Imported file already exists. Overwrite?', newline, 'Existing file: ',outfile];
-        overwrite = questdlg(msg, 'File already exists', 'Yes', 'No', 'No'); % default not to overwrite by users 
-    else
-        overwrite = 'No'; % default not to overwrite on Jenkins
-    end             
-    close gcf;
-else
-    overwrite = 'Yes';
-end;
-if strcmp(overwrite, 'No')
-    warning('Data discarded ...');
-    return;
+
+if pspm_overwrite(outfile, options)
+	save(outfile, 'data', 'fileinfo');
 else
-    save(outfile, 'data', 'fileinfo');
-end;
+	warning('Data discarded ...');

```

Comment:
[minor comment]
Would a naive user reading this warning be able to understand where it comes from and why?
Could it be extended with more information, while still being easy to read?

Reply:
Just to clarify: "Backroom" includes functions that cannot be accessed from the GUI - this is for development and legacy purposes.

Reply:
> [minor comment]
> Would a naive user reading this warning be able to understand where it comes from and why?
> Could it be extended with more information, while still being easy to read?

Yes.. Some warnings caused by overwriting are proposed in `pspm_overwrite` thus they are not repeated here. I am not the author of this script so I have not done any updates in this function but just applied the new APIs.